pallets/briefs/src/lib.rs

```diff
@@ -315,6 +316,22 @@ pub mod pallet {
             Self::deposit_event(Event::<T>::BriefEvolution(brief_id));
             Ok(())
         }
+
+        /// Extrinsic to cancel a brief
+        #[pallet::call_index(5)]
+        #[pallet::weight(<T as Config>::WeightInfo::cancel_brief())]
+        pub fn cancel_brief(origin: OriginFor<T>, brief_id: BriefHash) -> DispatchResult {
+            let who = ensure_signed(origin)?;
+            let brief = Briefs::<T>::get(brief_id).ok_or(Error::<T>::BriefNotFound)?;
```

Comment:
i need a test for all errors that youve specified here
BriefNotFound,
NotAuthorised